tests/manifold/spherical_manifold_03.output

```diff
@@ -1,3 +1,3 @@
 
-DEAL::Testing dim 2, spacedim 2
-DEAL::Testing dim 3, spacedim 3
+DEAL::1.57080 0.00000 0.00000
+DEAL::0.00000 1.57080 0.00000
```

Comment:
@bangerth I used the test of [#2763](https://github.com/dealii/dealii/issues/2763#issuecomment-231340365) to check this PR. It seems working.